x-pack/plugins/endpoint/server/routes/resolver/queries/events.ts

```diff
@@ -6,10 +6,11 @@
 import { ResolverQuery } from './base';
 import { JsonObject } from '../../../../../../../src/plugins/kibana_utils/public';
 
-export class RelatedEventsQuery extends ResolverQuery {
+export class EventsQuery extends ResolverQuery {
   protected legacyQuery(endpointID: string, uniquePIDs: string[], index: string): JsonObject {
```

Comment:
I'd like to modify this so it can only take a single unique_pid (or entity_id for the query function).  We can make the parameter type in the ResolverQuery class `string[] | string` and allow subclasses to specify whether they allow single values, arrays, or both.